packages/dataviews/src/dataviews-layouts/table/index.tsx

```diff
@@ -7,17 +7,13 @@ import clsx from 'clsx';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import {
-	Spinner,
-	__experimentalHStack as HStack,
-	__experimentalVStack as VStack,
-} from '@wordpress/components';
+import { Spinner } from '@wordpress/components';
 import { useEffect, useId, useRef, useState } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import SingleSelectionCheckbox from '../../components/dataviews-selection-checkbox';
+import DataViewsSelectionCheckbox from '../../components/dataviews-selection-checkbox';
```

Comment:
In `trunk`, we have `SingleSelectionCheckbox` and `BulkSelectionCheckbox`. With this change, we'd have `DataViewsSelectionCheckbox` and `BulkSelectionCheckbox`. Did you have something in mind when doing this change? If nothing changes in terms of behaviour, I'd prefer keeping the existing names, for clarity.

Reply:
I just followed the file name.

Reply:
For me it's confusing to import a default export from a file and the file name is different than the component name.

Reply:
Also within the file itself it's called `DataViewsSelectionCheckbox`

Reply:
> I agree, file names and components should be coherent. If we want to do something, we could do the opposite: rename the file to single-selection-checkbox.

I would be fine with that, but it involves a lot of changes (classnames...), we can do it in its own PR if needed.